tests/indices/vector_store/utils.py

```diff
@@ -63,8 +63,15 @@ def query(
 def get_pinecone_storage_context() -> StorageContext:
     # NOTE: mock pinecone import
```

Comment:
Super NIT: this comment is unclear to me - is it saying this is mocking the pinecone import or that we need to remember to go back and mock the pinecone import?

Reply:
Ah, I'll clarify (fwiw, I don't think I was the author of the original comment)

Reply:
Done